platform/macos/src/MGLMapView.mm

```diff
@@ -252,7 +252,7 @@ + (NSArray *)restorableStateKeyPaths {
     return @[@"camera", @"debugMask"];
 }
 
-- (void)commonInit {
+- (void)commonInit:(nullable NSString*)fontFamily {
```

Comment:
In terms of plumbing the value through mbgl, consider adding an overridable method to `mbgl::RendererFrontend` that returns a CTFontDescriptor. The Darwin implementation of MGLRenderFrontend can convert to and from NSFontDescriptor or UIFontDescriptor, using typedefs and conditional compilation to paper over differences between iOS and macOS.

Since CTFontDescriptor is itself platform-specific, consider wrapping it in a new `mbgl::FontDescriptor` class along the lines of `mbgl::RendererFrontend` before returning it to `mbgl::RendererFrontend`.